nois/chain.json

```diff
@@ -82,45 +76,53 @@
     "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nois/images/nois.png",
     "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nois/images/nois.svg"
   },
-  "peers": {
-    "seeds": [
-      {
-        "id": "b3e3bd436ee34c39055a4c9946a02feec232988c",
-        "address": "seeds.cros-nest.com:56656",
-        "provider": "Cros-Nest"
-      },
-      {
-        "id": "babc3f3f7804933265ec9c40ad94f4da8e9e0017",
-        "address": "seed.rhinostake.com:17356",
-        "provider": "RHINO"
-      },
-      {
-        "id": "ade4d8bc8cbe014af6ebdf3cb7b1e9ad36f412c0",
-        "address": "seeds.polkachu.com:17356",
-        "provider": "Polkachu"
-      },
-      {
-        "id": "72cd4222818d25da5206092c3efc2c0dd0ec34fe",
-        "address": "161.97.96.91:36656",
-        "provider": "AM Solutions"
-      },
-      {
-        "id": "20e1000e88125698264454a884812746c2eb4807",
-        "address": "seeds.lavenderfive.com:17356",
-        "provider": "Lavender.Five Nodes 🐝"
-      },
-      {
-        "id": "c8db99691545545402a1c45fa897f3cb1a05aea6",
-        "address": "nois-mainnet-seed.itrocket.net:36656",
-        "provider": "itrocket"
-      },
-      {
-        "id": "400f3d9e30b69e78a7fb891f60d76fa3c73f0ecc",
-        "address": "nois.rpc.kjnodes.com:15159",
-        "provider": "kjnodes"
-      }
-    ]
-  },
+  "persistent_peers": [
+    
```

Comment:
this isn't following the schema. I believe the 'seeds' and 'persistent_peers' array have to be inside a 'peers' object